silo-oracles/deploy/chainlink-v3-oracle/configs/sonic.json

```
@@ -136,5 +136,16 @@
         "normalizationDivider": 1e16,
         "normalizationMultiplier": 0,
         "invertSecondPrice": false
+    },
+    "PYTH_HARDCODE_wstkscUSD_frxUSD": {
+        "baseToken": "wstkscUSD",
+        "quoteToken": "frxUSD",
```

Isn't it returns scUSD ?
```suggestion
        "quoteToken": "scUSD",
```